tests/cases/eth2eth/scripts/test-tx

```diff
@@ -5,37 +5,75 @@ set -eu
 SCRIPT_DIR=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
 source ${SCRIPT_DIR}/util/relayer-util
 
-BALANCE_GO=${SCRIPT_DIR}/../../../chains/ethereum/scripts/cmd/balance/main.go
-TRANSFER_GO=${SCRIPT_DIR}/../../../chains/ethereum/scripts/cmd/transfer/main.go
+ICS20_CLI="ethereum-ics20-cli"
 
-SIMPLE_TOKEN_ADDRESS="0xff77D90D6aA12db33d3Ba50A34fB25401f6e4c4F"
+TMP_DIR=/tmp
+if ! which ${ICS20_CLI} > /dev/null 2>&1; then
+  if [ ! -d "${TMP_DIR}" ]; then
+    mkdir -p ${TMP_DIR}
+  fi
+  git clone https://github.com/datachainlab/ethereum-ics20-cli.git ${TMP_DIR}/ethereum-ics20-cli
+  cd ${TMP_DIR}/ethereum-ics20-cli
+  go install
```

Comment:
Is there any harm in running `go install github.com/datachainlab/ethereum-ics20-cli`?

Reply:
https://github.com/dongrie/yui-relayer-build/blob/use-ethereum-ics20-cli/go.mod#L129-L134
Because there was a replace in go.mod, the following error occurred.
![スクリーンショット 2023-05-12 11 20 42](https://github.com/datachainlab/yui-relayer-build/assets/126585618/ee7d0bfe-3cdd-4186-bafb-ed32659bd622)

Ref: https://stackoverflow.com/questions/69807151/go-install-github-com-dmacvicar-terraform-provider-libvirtlatest-shows-error